GVRf/Framework/framework/src/main/java/org/gearvrf/GVRViewManager.java

```diff
@@ -355,9 +361,10 @@ public void run() {
                     }
 
                     // Trigger event "onAfterInit" for post-processing of scene
-                    // graph after initialization.
+                    // graph after initialization. Also trigger "onInit" for the GVRContext.
                     getEventManager().sendEvent(mMain, IScriptEvents.class,
                             "onAfterInit");
+                    getEventManager().sendEvent(this, IContextEvents.class, "onInit", this);
```

Comment:
Should be replaced with ``getEventManager().sendEvent(GVRViewManager.this, IContextEvents.class, "onInit", GVRViewManager.this)``. ``this`` refers to the Runnable instance.